server/model/monitor.js

```diff
@@ -857,10 +857,19 @@ class Monitor extends BeanModel {
         if (tlsInfoObject && tlsInfoObject.certInfo && tlsInfoObject.certInfo.daysRemaining) {
             const notificationList = await Monitor.getNotificationList(this);
 
-            log.debug("monitor", "call sendCertNotificationByTargetDays");
-            await this.sendCertNotificationByTargetDays(tlsInfoObject.certInfo.daysRemaining, 21, notificationList);
-            await this.sendCertNotificationByTargetDays(tlsInfoObject.certInfo.daysRemaining, 14, notificationList);
-            await this.sendCertNotificationByTargetDays(tlsInfoObject.certInfo.daysRemaining, 7, notificationList);
+            let notifyDays = await setting("tlsExpiryNotifyDays");
+            if (notifyDays == null || !Array.isArray(notifyDays)) {
+                // Reset Default
+                setSetting("tlsExpiryNotifyDays", [ 7, 14, 21 ], "general");
```

Comment:
Just tried clean install of 1.17.0-beta, it seems that the default values is not working.

![image](https://user-images.githubusercontent.com/1336778/173487635-a43583f8-60bd-43f0-b88f-9c3d2487ef78.png)

Reply:
upgraded manually and got the same result, investigating.

Reply:
Actually me dumb, that part only runs the first time a HTTPS monitor is running. I guess should move it somewhere during server init.